opencga-core/src/main/java/org/opencb/opencga/core/config/QuotaConfiguration.java

```diff
@@ -0,0 +1,59 @@
+package org.opencb.opencga.core.config;
+
+public class QuotaConfiguration {
+
+    private int maxNumUsers;
```

Comment:
I'd make these `Integer` so they can be null in case we want to disable some quotas